docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,56 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+== Creating tags
+
+You can create custom tags for applications for applications that {ProductShortName} assesses or analyzes.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
+. Click *Create tag*, which opens a dialog.
+. In the *Name* field enter a unique name for the tag.
+. Click the *Tag category* field and select the category tag to associate with the field.
+. Click *Create*.
+
+[id="editing-tags_{context}"]
+= Editing tags
+
+You can edit defined tags.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab. A list of tag categories appears in the main pane.
+. Click the arrow to the left of the category name, opening a list of tags in the category.
+
+. Select a defined tag category and click the arrow to the left of the category name, opening a list of tags.
```

Comment:
- Step is duplicated
- In the original PR, this image comes here: docs/topics/images/mta-web-assessment-tags-03.png